e2e/rdctl.e2e.spec.ts

```diff
@@ -834,6 +835,23 @@ test.describe('Command server', () => {
         }
       });
     });
+    test('should list images based on current containerEngine', async() => {
+      const { stdout, stderr, error } = await rdctl(['list-settings']);
+      const settings = JSON.parse(stdout);
+      const containerEngine = settings.kubernetes.containerEngine;
+
+      expect(containerEngine).toMatch(/containerd|moby/);
+
+      if (containerEngine === 'moby') {
+        const output = await tool('docker', 'images');
+
+        expect(output).toMatch(/rancher.*library-busybox\s+[.0-9]+/);
```

Comment:
* I guess we always start with Kubernetes enabled, so the image should exist.
* Do we need to worry about some version of k3s no longer including this image? Can we get away with something shorter, like `rancher/` by itself?

Reply:
Actually, we can get by with the header only. If k8s is off, the images might not be present. I just wanted to verify that we can still run the container-engine's clients and get sane output.